docs/lib/datastore/other-methods.md

```diff
@@ -4,3 +4,4 @@ description: Other Amplify Datastore methods
 ---
 
 <inline-fragment platform="js" src="~/lib/datastore/fragments/js/other-methods.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/other-methods.md"></inline-fragment>
```

Comment:
Probably outside of the scope of this PR, but if the heading "Other methods" seems less than ideal to me.

Reply:
Yeah, I agree it's not the best.  I copied it from JS.  Maybe when iOS is ready to add their version and we can have one common file, we can also rethink the title then.